IBPSA/Fluid/Actuators/Dampers/MixingBox.mo

```diff
@@ -3,8 +3,15 @@ model MixingBox "Outside air mixing box with interlocked air dampers"
   extends IBPSA.Fluid.Actuators.BaseClasses.ActuatorSignal;
 
   replaceable package Medium =
-      Modelica.Media.Interfaces.PartialMedium "Medium in the component"
-    annotation (choicesAllMatching = true);
+    Modelica.Media.Interfaces.PartialMedium "Medium in the component"
+      annotation (choices(
+        choice(redeclare package Medium = IBPSA.Media.Air "Moist air"),
+        choice(redeclare package Medium = IBPSA.Media.Water "Water"),
+        choice(redeclare package Medium =
+            IBPSA.Media.Antifreeze.PropyleneGlycolWater (
+          property_T=293.15,
+          X_a=0.40)
+          "Propylene glycol water, 40% mass fraction")));
```

Comment:
@JayHuLBL : Why using an air damper with water or glycol? Please change. This could even be set as default to moist air.